CHANGELOG.md

```diff
@@ -26,27 +26,24 @@ CHANGELOG
 Swift Next
 ----------
 
-* [SR-8974][]:
+* [SR-4206][]:
 
-  Duplicate tuple element labels are no longer allowed, because it leads
-  to incorrect behavior. For example:
+  Overriden methods are no longer allowed to have a different generic 
```

Comment:
Note that they can have a different generic signature as long as it has fewer requirements than the base method (once we fix the thunking for that anyway :) ). I would phrase this as "A method override is no longer allowed to have a generic signature with requirements not imposed by the base method. For example, ..."

Reply:
Done :)